### PR TITLE
Add space in section 4.2 between type and Integer

### PR DESCRIPTION
**Pull request by ArjunSaili1**
Section 4.4 is missing a space between words 'type' and Integer. I've added a space. 

Screen Shot 2022-10-08 at 9 26 15 PM

### CODE REVIEW

File: 04-assorted-topics-in-java.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ Here, we did the following changes:
 2. Everywhere we used `Object`, we switched it with `T`.
 3. When we create an instance of `MyItem`, we specify that `T` should be `Integer`.
 
-Note the mapping in syntax between `MyItem<T>` and `MyItem<Integer>`! Everywhere we use `T` in our class, we can pretend it says `Integer` for this instance. Furthermore, we don't need to do any casting as we know `mi.item` will be of typeInteger since `mi` is of type `MyItem<Integer>`. We can make new `MyItem` objects with other types too, if we'd like!
+Note the mapping in syntax between `MyItem<T>` and `MyItem<Integer>`! Everywhere we use `T` in our class, we can pretend it says `Integer` for this instance. Furthermore, we don't need to do any casting as we know `mi.item` will be of type Integer since `mi` is of type `MyItem<Integer>`. We can make new `MyItem` objects with other types too, if we'd like!
```
Comment (jcal13) [inline 04-assorted-topics-in-java.md:186]: Oh, could you add the ` ` around Integer too? Then We can merge it in. Thanks for catching this.

**Review by jcal13 — CHANGES_REQUESTED** [review]
See the requested change. Thanks!

File: 04-assorted-topics-in-java.md (hunk 1 above)
Comment (ArjunSaili1) [reply 04-assorted-topics-in-java.md]: My pleasure! I've added the ` quotes as requested.

**Review by jcal13 — APPROVED** [review]
Perfect, thanks!

**Pull request merged**